workers/tests/unittests/test_ohsome_client.py

```diff
@@ -40,15 +42,107 @@ def setUp(self) -> None:
 
     def test_query_valid_response(self) -> None:
         with patch("httpx.AsyncClient.post", new_callable=AsyncMock) as mock_request:
-            mock_request.return_value = httpx.Response(200, content=self.valid_geojson)
+            mock_request.return_value = httpx.Response(
+                200,
+                content=self.valid_geojson,
+                request=httpx.Request("POST", "url"),
```

Comment:
I would suggest to use a valid URL here. I would even use the “correct” URL for your mocked result, e.g. `https://api.ohsome.org/v1/elements/geometry`.